az-sphere-os-sdk.yaml

```diff
@@ -0,0 +1,6 @@
+name: Azure Sphere Config Store
```

Comment:
What is this YAML file for?

Reply:
It is used to build the unit test.

Reply:
It allows us to build the tests using the OS SDK.